src/main/java/core/Game.java

```diff
@@ -677,16 +681,34 @@ public static void main(String[] args) {
         /* 3. Set up players for the game */
         ArrayList<AbstractPlayer> players = new ArrayList<>();
 
-//        players.add(new RandomPlayer());
-        players.add(new RMHCPlayer());
-        players.add(new MCTSPlayer());
-        players.add(new OSLAPlayer());
+        MCTSParams params1 = new MCTSParams();
```

Comment:
I'm guessing this is for some experiments, but it makes the main entry point to running the framework complex and foreign to someone just getting started with spelled-out parameters for MCTS. Can we keep this main method super small and tidy, and maybe have other run classes/methods that do fancier things?

Reply:
Agreed. Will simplify and tidy up.